tools/parallel_UT_rule.py

```diff
@@ -1873,6 +1874,7 @@
     'test_dataloader_unkeep_order',
     'test_parallel_executor_profiler',
     'test_correlation',
+    'test_mkldnn_conv_affine_channel_fuse_pass',
```

Comment:
I don't see mkldnn pass tests here. And it looks for tests running on GPU. Please confirm if adding mkldnn tests here is necessary

Reply:
You're probably right. I left it because the old version added it here, but I'll try to remove it.